Idno/Entities/User.php

```diff
@@ -610,7 +610,7 @@ function addFollowing($user)
                     }
 
                     $acl->addMember($user->getUUID());
-                    $acl->save();
+                    $acl->save(true);
```

Comment:
small preference for something like `$acl->save(overrideAccess: true)` to make this more readable.  (for all usages)